components/SectionTitle.tsx

```diff
@@ -16,16 +16,21 @@ export default function SectionTitle(props: Props) {
   const router = useRouter()
   const path = router.asPath.split('#')[0]
   const id = `section-${props.index}`
+  const generated = new Date()
 
   return (
-    <h3
-      className="mb-8 text-2xl font-bold border-b-2 border-black pb-2"
-      id={id}
-    >
-      {props.icon && <span className="mr-2">{props.icon}</span>}
-      <Link href="/reports/[name]" as={`${path}#${id}`}>
-        <a>{props.children}</a>
-      </Link>
-    </h3>
+    <div>
+      <h3
+        className="mb-8 text-2xl font-bold border-b-2 border-black pb-2"
+        id={id}
+      >
+        {props.icon && <span className="mr-2">{props.icon}</span>}
+        <Link href="/reports/[name]" as={`${path}#${id}`}>
+          <a>{props.children}</a>
+        </Link>
+      </h3>
+      <div>Generated: {generated.toLocaleString()}</div>
```

Comment:
Moved this and de-emphasized it visually:

<img width="451" alt="Screen Shot 2020-09-18 at 11 29 00 AM" src="https://user-images.githubusercontent.com/412282/93616020-2a6c9a00-f9a2-11ea-88e9-c2e33abf8e71.png">